keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -229,20 +306,37 @@ https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
+If strict decoding fails, proxy fall back to a lenient decoder, ensuring  that
+rollouts and rollbacks remain unaffected in case of a malformed configuration.
+
+Running workloads are not affected.
+
 ###### What specific metrics should inform a rollback?
 
+N/A
+
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
+N/A
+
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
+The API field `PortRange` will be removed. 
```

Comment:
It's not being "removed" in the sense this question is asking; it will still exist in `v1alpha1.KubeProxyConfiguration`, which isn't going away.